### ISSUE TITLE
Theme switch not working after MSI installation

### ISSUE THREAD

**Issue by leihaung-ica** [ironmansoftware/powershell-universal#3888]
### Version

5.0.9

### Severity

Low

### Environment

msi

### Steps to Reproduce

Install 5.0.9 with msi installer in windows server 2022.
setup Windows auth

the default admin works, no problem switch to dark theme,
but when using AD account with SSO, the account cannot switch theme, the switcher flipped to dark theme, but the theme did not change.

but if I make a new instanse with zip file and IIS, it works.

### Expected behavior

```console
theme switch should work
```

### Actual behavior

```console
theme switch does not work.
```

### Screenshots/Animations

![image](https://github.com/user-attachments/assets/2c722c96-8e7a-4d60-a2a0-1ae1f89ca353)

**Comment by leihaung-ica** [comment]
more default can be found in https://forums.ironmansoftware.com/t/dark-theme-not-working-in-5-0-9/11479

**Comment by adamdriscoll** [comment]
@leihaung-ica Can you check the identities page? I'm wondering if you have duplicate identities for your user. We fixed an issue in some version where the database was not using the correct collation for the identity name column and you could be experiencing and issue where you have multiple identities of the same name. You would need to then delete the duplicate.

**Comment by leihaung-ica** [comment]
I will setup a new server from scratch and check it.

**Comment by leihaung-ica** [comment]
![image](https://github.com/user-attachments/assets/c79f92c8-9e2a-4e82-b9ab-b6801fe57a09)

I setup a brand new server, installed msi, 5.0.13 this time, using postgresql, enabled Windows Authentication.
logged in and went to My Identify page

**Comment by leihaung-ica** [comment]
you mentioned database collation, which one should I use for postgresql?
below is ours
![image](https://github.com/user-attachments/assets/229b4411-5934-4a5b-8836-4073b5d0023c)

**Comment by adamdriscoll** [comment]
@leihaung-ica - I think that collation should be fine but can you please check the log for an error message related to the My Identity page?

**Comment by leihaung-ica** [comment]
I went to log folder, saw the error, some blazer stuff, then I thought... well, I just clear every log files and recreate the error.
But once I turn off the windows service and removed the logs, restarted the service.
The portal lost it's license, I uploaded the dev license, refresh and test again...
the theme switch works ... I can't get back the error ... lol

I didn't remember if I upgraded this instance from 5.0.12 to 5.0.13 or if it's 5.0.13 from the start.

But I assume this is not an issue anymore if I can't not recreat again ...
